### ISSUE TITLE
Windows installer file type association problematic

### ISSUE THREAD

**Issue by linquize** [mb21/panwriter#31]
Windows installer:
User cannot opt-out file type association.
File type association in Windows registry is problematic.
After installing panwriter from installer,
Explorer -> Right Click -> New -> Text Document is removed because some registry values are wrong.

I want to opt-out file type association.

**Comment by mb21** [comment]
Did you verify that that would fix it?

I'm not a Windows-user... but sounds also like something that should somehow be configurable in the registry or somewhere?

**Comment by linquize** [comment]
This is not to "fix", but to produce one more output:  **portable** executable. The problematic **nsis setup** is also produced. 
Let user to choose to download **portable**.

**Comment by mb21** [comment]
Yes, but have you verified that the portable version doesn't have this problem? I don't understand why it would be different...?

**Comment by linquize** [comment]
NSIS setup writes file assoxiations to Windows registry,  because you put `fileAssociations` in `package.json`. electron-builder will generate write registry code in NSIS script. 

Portable exe extracts files to a temporary directory before every run. It does not write to Windows registry.
Therefore, portable exe does not have such problem

**Comment by mb21** [comment]
I'll take a look for the next release... meanwhile, you can also [build it yourself](https://github.com/mb21/panwriter#develop)...
